guides/CODING-GUIDELINES.md

```diff
@@ -0,0 +1,39 @@
+# Coding guidelines
```

Comment:
I'm wondering if this file should be on the actual website? https://design.cms.gov/guidelines/code-conventions/